themes/src/main/resources/theme/base/login/resources/js/webauthnAuthenticate.js

```diff
@@ -0,0 +1,78 @@
+import { base64url } from "../../../common/keycloak/node_modules/rfc4648/lib/rfc4648.js";
+
+export function authenticateByWebAuthn(input) {
+    if (!input.isUserIdentified) {
+        doAuthenticate([], input.challenge, input.userVerification, input.rpId, input.createTimeout, input.errmsg)
+            .then(result => returnSuccess(result))
+            .catch(err => returnFailure(err));
+        return;
+    }
+    checkAllowCredentials(input.challenge, input.userVerification, input.rpId, input.createTimeout, input.errmsg);
+}
+
+function checkAllowCredentials(challenge, userVerification, rpId, createTimeout, errmsg) {
+    const allowCredentials = [];
+    const authn_use = document.forms['authn_select'].authn_use_chk;
```

Comment:
Let's keep the casing of variables consistent, even if it's different in the DOM.
```suggestion
    const authnUse = document.forms['authn_select'].authn_use_chk;
```

Reply:
I fixed the codes as you suggested.